akka-actor-typed-tests/src/test/scala/akka/actor/typed/pubsub/LocalPubSubSpec.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2009-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed.pubsub
+
+import akka.actor.testkit.typed.scaladsl.LoggingTestKit
+import akka.actor.testkit.typed.scaladsl.LogCapturing
+import akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit
+import org.scalatest.wordspec.AnyWordSpecLike
+import scala.concurrent.duration._
+
+class LocalPubSubSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike with LogCapturing {
+
+  "A pub-sub topic running locally" must {
+
+    "publish to all local subscriber actors of a topic" in {
+      val fruitTopic =
+        LoggingTestKit.debug("Topic list updated").expect {
+          testKit.spawn(Topic[String]("fruit"))
```

Comment:
would it be useful for this to be part of the API? An ack once registered with the receptionist? 

Or is that not actually useful as just because you've received the listing back doesn't mean other nodes have

Reply:
Yeah, it doesn't really mean anything useful except for possibly in this test and communication is always local so message won't be lost.